jekyll/_cci2/contexts.md

```diff
@@ -48,8 +48,51 @@ workflows:
+
+The default security group is `organization` and enables any member of the org to use the context.
+
+## Runnning Workflows with a Secure Context
+
+To invoke a workflow that uses a secure context, a user must be a member of one of the security groups for the context or the workflow will fail with the status of `Unauthorized`. If you add a context to your workflow and you are **not** a member of any of the context groups, the job will fail as `Unauthorized`.
+
+**Note:** Bitbucket repositories do **not** provide an API that allows CircleCI contexts to be secured, only GitHub hosted projects or projects that use GitHub Enterprise with CircleCI include the ability to limit contexts with security groups.
+
+### Limit a Context to a Security Group or Groups
+
+You must be an Org Admin to complete the following task.
+
+1. Navigate to Organization Settings > Contexts page in the CircleCI app. The list of contexts appears. The default security group is `organization` and allows all users in the org to invoke jobs with that context.
+2. Click the Create Context button if you will use a new context or click the name of an existing context.
+3. Click the Add Group link. The Add Groups dialog box appears.
+4. Select GitHub teams or LDAP groups to add to the context and click the Add button. Use of the context is now limited to the selected groups.
+5. Click Add Environment Variables to add environment variables to the context if none exist and click the Add button. Use of the environment variables for this context is now limited to members of the security groups.
+6. Navigate to Organization Settings > Contexts in the CircleCI app. The security groups appear in the Security column for the context.
+
+Only members of the selected groups may now use the context in their workflows or add or remove environment variables for the context.
+
+## Removing Groups from Contexts
+
+To make a context available only to the admins of the org, you may remove all of the groups associated with a context. All other users will lose access to that context.
+
+### Adding and Removing Users from Teams and Groups
+
+CircleCI syncs GitHub team and LDAP groups every four hours. If a user is added or removed from a GitHub team or LDAP group, it will take up to four hours to update the CircleCI records and remove access to the context.
+
+### Adding and Removing Environment Variables from Secure Contexts
+
+Addition and deletion of environment variables from a secure context is limited to members of the context groups.
+
+### Approving Jobs that use Secure Contexts
+
+Manual invocation of a job that is `type: approval` is limited to members of the context groups.
+
 ## Deleting a Context
 
+If the context is secured with a group other than `organization`, you must be an org admin to complete this task.
```

Comment:
oh that is not true.. Any one who is member of the group can delete the context.

Reply:
Ok, thanks,  changed to "If the context is secured with a group other than `organization`, you must be a member of the security group to complete this task."